imod/mf6/buy.py

```diff
@@ -157,6 +157,9 @@ def update_transport_models(self, new_modelnames: Sequence[str]):
         transport_model_names = self.get_transport_model_names()
         if not len(transport_model_names) == len(new_modelnames):
             raise ValueError("the number of transport models cannot be changed.")
+        for i in range (0, len(transport_model_names)):
```

Comment:
Can be simplified to:

```suggestion
        for i in range (len(transport_model_names)):
```

Or even better:

```suggestion
        for modelname, new_modelname in zip(transport_model_names, new_modelnames):
            if not modelname in new_modelname:
```